unit_testing/cli/test_cli_sct_deepseg_sc.py

```diff
@@ -12,3 +15,10 @@ def test_sct_deepseg_sc_backwards_compat(script_runner):
     logger.debug(f"{ret.stderr}")
     assert ret.success
     assert ret.stderr == ''
+
+
+def test_sct_deepseg_sc_o_flag(tmp_path):
+    command = """-i sct_testing_data/t2/t2.nii.gz -c t2 -o""" + str(os.path.join(str(tmp_path), 'test_seg.nii.gz'))
+    sct_deepseg_sc.main(command.split())
+    assert os.path.isfile(os.path.join(str(tmp_path), 'test_seg.nii.gz'))
```

Comment:
`tmp_path` returns a `pathlib.Path` object, which [has some useful properties](https://realpython.com/python-pathlib/). You can use `/` to combine paths instead of `os.path.join`! And, when combined with f-strings (`f""`), you don't need any of the `str()` conversion, so it can be written in a short way:

```suggestion
    tmp_filepath = tmp_path/'test_seg.nii.gz'
    command = f"-i sct_testing_data/t2/t2.nii.gz -c t2 -o {tmp_filepath}"
    sct_deepseg_sc.main(command.split())
    assert os.path.isfile(tmp_filepath)
```

Reply:
This same refactor could apply to the other tests, too. :)

Reply:
I went with argv so no need for formatted string ;)